sample-input/gradients/one-directional/one-directional-cmfd.py

```diff
@@ -0,0 +1,155 @@
+import numpy
+from openmoc import *
+import openmoc.log as log
+import openmoc.plotter as plotter
+from openmoc.options import Options
+
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+
+log.py_printf('TITLE', 'Simulating a one group homogeneous, one directional'
+    ' gradient...')
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating materials...')
+
+basic_material = Material(name='1-group infinite medium')
+basic_material.setNumEnergyGroups(1)
+basic_material.setSigmaA(numpy.array([0.069389522]))
+basic_material.setSigmaF(numpy.array([0.0414198575]))
+basic_material.setNuSigmaF(numpy.array([0.0994076580]))
+basic_material.setSigmaS(numpy.array([0.383259177]))
+basic_material.setChi(numpy.array([1.0]))
+basic_material.setSigmaT(numpy.array([0.452648699]))
+
+
+###############################################################################
+###########################   Creating Surfaces   #############################
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating surfaces...')
+
+L = 100.0
```

Comment:
Can we move ``L`` as a runtime parameter to the top block "Main Simulation Paramters" in the input file? And perhaps rename it to something more intuitive such as ``length`` or at least comment it in each input file.